codespell_lib/_codespell.py

```diff
@@ -35,8 +35,16 @@
 
 # Users might want to link this file into /usr/local/bin, so we resolve the
 # symbolic link path to the real path if necessary.
-default_dictionary = os.path.join(os.path.dirname(os.path.realpath(__file__)),
-                                  'data', 'dictionary.txt')
+_data_root = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+_builtin_dictionaries = (  # name, desc, name, should be in aspell
+    ('clear', 'for unambiguous errors', '', True),
+    ('rare', 'for rare but valid words', '_rare', True),
+    ('informal', 'for informal words', '_informal', True),
+    ('code', 'for words common to code and/or mathematics', '_code', True),
+    ('names', 'for proper names', '_names', False),
+    ('gb_to_us', 'for corrections from GB to US', '_gb_to_us', True),
```

Comment:
This should probably be explicit as 'for corrections from en-GB to en-US' to match https://en.wikipedia.org/wiki/Language_localisation#Language_tags_and_codes

Also if we use the key `to_en_US` or `to_en_us` if you want to keep it lower case, it gives people the option to potentially automate their choice using stuff like this:
https://en.wikipedia.org/wiki/Locale_(computer_software)#POSIX_platforms

And logically any non-US spellings should be in there, if there was say an Australian way of saying something. Although that would break my suggest ability elsewhere to reverse it.

Reply:
If it's not clear what to do here we should just remove this dict and add it back later once we have a clear plan I think. For now I'm just going to change the wording